app/renderer/components/Inspector/ElementLocator.js

```diff
@@ -29,38 +56,37 @@ class ElementLocator extends Component {
       locatorTestValue,
       setLocatorTestStrategy,
       locatorTestStrategy,
+      driver,
       t,
     } = this.props;
 
-    const locatorStrategies = [
-      ['id', 'Id'],
-      ['xpath', 'XPath'],
-      ['name', 'Name'],
-      ['class name', 'Class Name'],
-      ['accessibility id', 'Accessibility ID'],
-      ['-android uiautomator', 'UIAutomator Selector (Android UiAutomator2)'],
-      ['-android datamatcher', 'DataMatcher Selector (Android Espresso)'],
-      ['-android viewtag', 'Android View Tag (Android Espresso)'],
-      ['-ios predicate string', 'Predicate String (iOS)'],
-      ['-ios class chain', 'Class Chain (iOS)'],
-    ];
-
-    return <div>
-      <Row>
-        {t('locatorStrategy')}
-        <Select className={InspectorStyles['locator-strategy-selector']}
-          onChange={(value) => setLocatorTestStrategy(value)}
-          value={locatorTestStrategy}>
-          {locatorStrategies.map(([strategyValue, strategyName]) => (
-            <Option key={strategyValue} value={strategyValue}>{strategyName}</Option>
+    return <Row justify="center">
+      <Radio.Group buttonStyle="solid"
+        className={InspectorStyles.locatorStrategyGroup}
+        onChange={(e) => setLocatorTestStrategy(e.target.value)}
+        defaultValue={locatorTestStrategy}
+      >
+        <Row justify="center">
```

Comment:
I think this row should have some kind of label like 'Strategy' in case someone isn't super familiar with what these are yet

Reply:
Added back the original label